vector/src/main/java/im/vector/app/features/createdirect/CreateDirectRoomViewModel.kt

```diff
@@ -124,7 +124,7 @@ class CreateDirectRoomViewModel @AssistedInject constructor(
                     }
 
             val result = runCatchingToAsync {
-                if (vectorFeatures.shouldStartDmOnFirstMessage()) {
+                if (vectorPreferences.isDeferredDmEnabled()) {
```

Comment:
Just a remark as I am seeing this now: can you check that analytics will be sent when the room will be created?
If the feature is disabled, it's sent in the `else` block.

Reply:
Good remark, done https://github.com/vector-im/element-android/pull/7180/commits/fa8b56b1ad0790f73a09661203a2f78b21a8d72d
I did not find a better place to add it